src/merge.py

```
@@ -807,277 +401,196 @@ def main():
 
-    if TELEGRAM_COMMAND_STATUS in telegram_commands:
-        send_telegram_message(_build_status_message(state, today, skip_days))
+    #region ICLOUD_CALENDAR_LOAD
+    print_step(TAG_CALENDAR_MERGE, "loading iCloud calendar events...", False)
+    calendar_service = icloud_service.calendar
+    try:
+        calendars = calendar_service.get_calendars()
+        calendar_guid = next((c.get("guid") for c in calendars if c.get("guid")), None)
+    except Exception as err:
+        term.print_failed()
+        raise RuntimeError("Unable to fetch calendars") from err
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)